redisvl/vectorize/text/vertexai.py

```diff
@@ -0,0 +1,142 @@
+from typing import Callable, Dict, List, Optional
+
+from tenacity import retry, stop_after_attempt, wait_random_exponential
+from tenacity.retry import retry_if_not_exception_type
+
+from redisvl.vectorize.base import BaseVectorizer
+
+
+class VertexAITextVectorizer(BaseVectorizer):
```

Comment:
annoying, I know, but I like ``Vertex`` or ``Google``. Just seems cleaner

Reply:
Hmmm I would disagree -- "Vertex" isn't a thing (see naming convention here: https://python.langchain.com/docs/integrations/llms/google_vertex_ai_palm) and "Google" is too vague. Just like OpenAI, I think we need to call it by it's name, even if verbose.